rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -158,25 +237,25 @@ trait NodeInfoServiceCached extends NodeInfoService  with Loggable with CachedRe
    *
    * attributes is the list of attributes needed in returned entries
    */
-  def getNodeInfoEntries(con: RoLDAPConnection, attributes: Seq[String]): Seq[LDAPEntry]
+  def getNodeInfoEntries(con: RoLDAPConnection, attributes: Seq[String], status: InventoryStatus): Seq[LDAPEntry]
 
   /*
    * Our cache
    */
-  private[this] var nodeCache = Option.empty[Map[NodeId, (LDAPNodeInfo, NodeInfo, Node)]]
+  private[this] var nodeCache = Option.empty[Map[NodeId, (LDAPNodeInfo, NodeInfo)]]
```

Comment:
I think it's the answer on my firt comment on the ldap schema !! (or i don't understand anything!)